newIDE/app/src/stories/componentStories/ProfileDetails.stories.js

```diff
@@ -0,0 +1,44 @@
+// @flow
+import * as React from 'react';
+import { action } from '@storybook/addon-actions';
+
+import muiDecorator from '../ThemeDecorator';
+import paperDecorator from '../PaperDecorator';
+
+import ProfileDetails from '../../Profile/ProfileDetails';
+import { indieUserProfile } from '../../fixtures/GDevelopServicesTestData';
+import {type Profile as ProfileType} from '../../Utils/GDevelopServices/Authentication'
+
+export default {
+  title: "ProfileDetails",
+  component: ProfileDetails,
+  decorators: [paperDecorator, muiDecorator],
+  argTypes: {
+    profile: {
+      control: { type: 'radio' },
+      options: ['Complete profile', 'Without username nor bio'],
+      defaultValue: 'Complete profile',
+      mapping: {
+        'Complete profile': indieUserProfile,
+        'Without username nor bio': {
```

Comment:
This is not typed and could break if one day username/description is not part anymore of the type/renamed. Either make a `indieWithUsernameAndDescriptionUserProfile` or add a `({ ... }): Profile` around this so that we're sure the type stays correct in the future :)